spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -169,6 +169,46 @@ cf start dataflow-server
 
 Alternatively, you can run the Admin application locally on your machine which is described in the next section.
 
+[[sample-manifest-teample]]
+==== Sample Manifest Template
+
+As an alternative to setting environment variables via `cf set-env` command, you can curate all the relevant env-var's
+in `manifest.yml` file and use `cf push` command to provision the server.
+
+Following is a sample template to provision the server on PCFDev.
+
+[source,bash,subs=attributes]
+----
+---
+applications:
+- name: <PREFERRED NAME OF THE SERVER APP>
```

Comment:
Seems like the `<` need to be escaped somehow? Don't render well in GH, at least